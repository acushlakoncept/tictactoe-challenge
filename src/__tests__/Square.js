import { func } from 'prop-types';
import React from 'react';
import renderer from 'react-test-renderer';
import Square from '../component/Square';

it('renders Square correctly', () => {
  const tree = renderer.create(<Square value={1} handleClick={func} />);
  expect(tree).toMatchSnapshot();
});
