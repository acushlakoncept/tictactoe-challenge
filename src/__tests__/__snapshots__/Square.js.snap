// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Square correctly 1`] = `
<button
  className="square btn-light 1 btn btn-primary"
  disabled={false}
  onClick={[Function]}
  type="button"
>
  1
</button>
`;
